tests/ConfiguratorTest.php

```diff
@@ -24,15 +27,6 @@ public function testItRequiresAllowedHosts()
 		$this->assertEquals(500, http_response_code());
 	}
 
-	public function testItRequiresCacheExpirationHours()
-	{
-		putenv('CACHE_EXPIRATION_HOURS'); // Clear env var
-		putenv('ALLOWED_HOSTS=example.com,anotherexample.com');
-		$this->setExpectedException(\LogicException::class, 'The CACHE_EXPIRATION_HOURS configuration is missing.');
-		Configurator::validateEnvironment();
-		$this->assertEquals(500, http_response_code());
-	}
-
 	public function testItConfiguresConfigurator()
```

Comment:
This is quite the fun**ny**ction name.  If this test evaluates - as I think it does - whether request values get set on configurations instantiated with those requests, it might be good to at least acknowledge this code as boilerplate construction.